app/javascript/components/atoms/colors.js

```diff
@@ -0,0 +1,35 @@
+export const COLORS = {
```

Comment:
here's the colors constants I pulled in. figured we could generalize to styles if we need to at some point, but for now colors seem like enough